proto/info/info.proto

```diff
@@ -1135,7 +1134,6 @@ message ZInfoMsg {
     ZInfoHardware hwinfo = 18;
     ZInfoLocation locinfo = 19;
     ZInfoPatchEnvelope patchInfo = 20;
-    ZInfoOpaqueAppInstanceStatus opaqueAppInstStatus = 21;
```

Comment:
I think it would be good to add a comment that 21 was previously defined (even though it hasn't been used) and is now deprecated.

Reply:
Added deprecated comment

Reply:
https://protobuf.dev/programming-guides/proto2/

According to this `ZInfoOpaqueAppInstanceStatus opaqueAppInstStatus = 21 [deprecated=true];` is the suggested way to deprecate existing fields. Can we start doing that?

Reply:
@gkodali-zededa Thanks for that pointer. 
But reading that document it seems to say that reserved 21;
is the better approach since deprecated doesn't prevent it from being used (but might result in protoc warnings for some languages.)

@uncleDecart want to try with reserved for this and the field above?
I'm not sure whether there should be a single reserved statement per message with multiple values, or if you can have multiple reserved statements.

Reply:
@eriknordmark I added reserved field for things that I've deleted, pb files generated, so I assume it works